spec/ndx-patterned-ogen.extensions.yaml

```diff
@@ -1,4 +1,5 @@
 groups:
+
```

Comment:
Also added spacing between type definitions here in the spec to make them easier to visually identify